File: xamarin/Samples/Showcase-Android/LoginActivity.cs
using Android.App;
using Android.OS;
using Android.Widget;
using Com.Liferay.Mobile.Screens.Auth.Login;
using Com.Liferay.Mobile.Screens.Context;

namespace ShowcaseAndroid
{
    [Activity(Label = "LoginActivity")]
    public class LoginActivity : Activity, ILoginListener
    {
        private LoginScreenlet loginScreenlet;

        protected override void OnCreate(Bundle savedInstanceState)
        {
            base.OnCreate(savedInstanceState);
            SetContentView(Resource.Layout.LoginView);

            loginScreenlet = (LoginScreenlet)FindViewById(Resource.Id.login_screenlet);
            loginScreenlet.Listener = this;

            setDefaultValues();
        }

        /* LoginListener */

        public void OnLoginFailure(Java.Lang.Exception p0)
        {
            Toast.MakeText(this, "Login failed", ToastLength.Short).Show();
        }

        public void OnLoginSuccess(User p0)
        {
            Toast.MakeText(this, "Login successful", ToastLength.Short).Show();
        }

        /* Private methods */

        void setDefaultValues()
        {
            EditText login = (EditText)loginScreenlet.FindViewById(Resource.Id.liferay_login);
            EditText password = (EditText)loginScreenlet.FindViewById(Resource.Id.liferay_password);

            login.SetText("test@liferay.com", TextView.BufferType.Editable);
            password.SetText("test1", TextView.BufferType.Editable);
        }
    }
}

File: android/samples/pushnotifications/src/main/java/com/liferay/mobile/pushnotifications/activities/LoginActivity.java
package com.liferay.mobile.pushnotifications.activities;

import android.content.Intent;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.widget.EditText;
import com.liferay.mobile.pushnotifications.R;
import com.liferay.mobile.screens.auth.login.LoginListener;
import com.liferay.mobile.screens.auth.login.LoginScreenlet;
import com.liferay.mobile.screens.context.User;

public class LoginActivity extends AppCompatActivity implements LoginListener {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        LoginScreenlet loginScreenlet = findViewById(R.id.login_screenlet);
        loginScreenlet.setListener(this);

        setDefaultValuesForUserAndPassword();
    }

    @Override
    public void onLoginSuccess(User user) {
        startActivity(new Intent(this, NotificationsActivity.class));
    }

    @Override
    public void onLoginFailure(Exception e) {

    }

    @Override
    public void onAuthenticationBrowserShown() {

    }

    private void setDefaultValuesForUserAndPassword() {
        EditText login = findViewById(R.id.liferay_login);
        login.setText(R.string.default_user);

        EditText password = findViewById(R.id.liferay_password);
        password.setText(R.string.default_password);
    }
}
